api/wallet_api-v0.yml

```diff
@@ -101,16 +101,13 @@ paths:
             $ref: "#/definitions/Error"
     put:
       operationId: restWalletImport
-      parameters:
-        - in: body
-          name: body
-          required: true
-          x-nullable: false
-          schema:
-            $ref: "#/definitions/Wallet"
       produces:
         - application/json
       responses:
+        "200":
+          description: New wallet created.
+          schema:
+            $ref: "#/definitions/Wallet"
```

Comment:
Why do you need to give out something that was provided as input? 
Do you intend to provide a secure retrieval or outsourced backup mechanism (using the password so as not to provide the private key in clear)?